tests/unit/s2n_server_key_share_extension_test.c

```diff
@@ -379,69 +374,48 @@ int main(int argc, char **argv)
         } while (i < ecc_pref->count);
     }
 
-    /* Test s2n_extensions_server_key_share_send with supported curve not in s2n_ecc_preferences list selected */
+    /* Test s2n_server_key_share_extension.send with supported curve not in s2n_ecc_preferences list selected */
     if (s2n_is_evp_apis_supported()) {
         struct s2n_connection *conn;
-
         EXPECT_NOT_NULL(conn = s2n_connection_new(S2N_SERVER));
         EXPECT_NOT_NULL(conn->config);
-        /* Explicitly set the ecc_preferences list to contain the curves p-256 and p-384 */
-        EXPECT_SUCCESS(s2n_config_set_cipher_preferences(conn->config, "20140601"));
-
-        const struct s2n_ecc_preferences *ecc_pref = NULL;
-        EXPECT_SUCCESS(s2n_connection_get_ecc_preferences(conn, &ecc_pref));
-        EXPECT_NOT_NULL(ecc_pref);
 
-        /* x25519 is present in s2n_all_supported_curves_list but not in the "default" list */
+        /* x25519 is supported, but not present in all curve preference lists */
```

Comment:
all curve preference lists seems a little vague, what we are trying to convey here is that x25519 is supported by s2n but `conn->config->security_policies->ecc_preferences` is set to a ecc_preferences list without "x25519" in it.